### PR TITLE
Add a VS Code task to run `sync-files.py`

### PR DESCRIPTION
**Pull request by dbartol**
If you're developing one of the libraries that has muiltiple copies auto-generated by `sync-files.py`, you can now run `sync-files.py --latest` by going to the `Terminal | Run Task...` menu in VS Code and selecting the `Sync Identical Files` task. You can set a keyboard binding to run this task for quicker access.

### CODE REVIEW

File: .vscode/tasks.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    // See https://go.microsoft.com/fwlink/?LinkId=733558
```
Comment (jbj) [inline .vscode/tasks.json:2]: I found this article to be too long to skim quickly when I just want to learn how to _invoke_ the task, not maintain it. Can you add a comment at the top of this file about the recommended way of running this task from VSCode? It worked for me to type Ctrl+P and typing "task" followed by a space, but that magic incantation seemed like a very hidden feature. Is the task supposed to show up when pressing Ctrl+Shift+B?

Comment (dbartol) [reply .vscode/tasks.json:5]: The existing link is added when you create a skeleton `tasks.json`, so it's intended to show you how to modify the file, not how to run tasks. I've added a brief comment about how to run tasks at the top, but I'm not sure that anyone who doesn't know how to run tasks in VS Code is likely to wind up looking at `tasks.json`. I've added a section to `README.md` on using Visual Studio Code to work in the repo, including information about `tasks.json` and about installing the CodeQL extension.

Comment (dbartol) [reply .vscode/tasks.json:5]: I also added it to the "build" task group, which means it should now show up when you hit Ctrl+Shift+B. I didn't add it as the _default_ build task, though, to leave room for build tasks for extractors.

Comment (dbartol) [reply .vscode/tasks.json:5]: BTW, did the task actually work when you did run it? I wasn't able to test on MacOS.

File: .vscode/tasks.json [hunk 2]
```diff
@@ -12,6 +15,7 @@
                 "config/sync-files.py",
                 "--latest"
             ],
+            "group": "build",
```
Comment (jbj) [inline .vscode/tasks.json:18]: With this change, the task shows up in Ctrl+Shift+B.

File: .vscode/tasks.json (hunk 1 above)
Comment (jbj) [reply .vscode/tasks.json:5]: I'm no longer on Mac. My new laptop runs Linux, and it works there.

**Pull request merged**